pending/xnview.xml

```diff
@@ -0,0 +1,139 @@
+    <action command="ini" search="file" path="$$ProgramFiles$$\XnView\xnview.ini" name="DirName_0"/>
+    <action command="ini" search="file" path="$$ProgramFiles$$\XnView\xnview.ini" name="DirName_1"/>
+    <action command="ini" search="file" path="$$ProgramFiles$$\XnView\xnview.ini" name="DirName_2"/>
+    <action command="ini" search="file" path="$$ProgramFiles$$\XnView\xnview.ini" name="DirName_3"/>
+    <action command="ini" search="file" path="$$ProgramFiles$$\XnView\xnview.ini" name="DirName_4"/>
+    <action command="ini" search="file" path="$$ProgramFiles$$\XnView\xnview.ini" name="DirName_5"/>
+    <action command="ini" search="file" path="$$ProgramFiles$$\XnView\xnview.ini" name="DirName_6"/>
+    <action command="ini" search="file" path="$$ProgramFiles$$\XnView\xnview.ini" name="DirName_7"/>
+    <action command="ini" search="file" path="$$ProgramFiles$$\XnView\xnview.ini" name="DirName_8"/>
+    <action command="ini" search="file" path="$$ProgramFiles$$\XnView\xnview.ini" name="DirName_9"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="FileName_0"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="FileName_1"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="FileName_2"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="FileName_3"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="FileName_4"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="FileName_5"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="FileName_6"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="FileName_7"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="FileName_8"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="FileName_9"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="DirName_0"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="DirName_1"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="DirName_2"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="DirName_3"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="DirName_4"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="DirName_5"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="DirName_6"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="DirName_7"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="DirName_8"/>
+    <action command="ini" search="file" path="%WinDir%\xnview.ini" name="DirName_9"/>
+    <action command="winreg" path="HKCU\Software\XnView\Start\FileName_*"/>
+    <action command="winreg" path="HKCU\Software\XnView\Start\DirName_*"/>
```

Comment:
Sorry, `command="winreg"` does not support wildcards

Reply:
Fixed!